client/src/endpoint/trade.rs

```diff
@@ -359,7 +359,7 @@ mod all_trades_tests {
 ///
 /// // Place the start time in the past so we capture it.
 /// let agg = trade::Aggregations::new(base, counter)
-///     .with_start_time(now - 3_000_000_000)
+///     .with_start_time(now - 50_000_000_000)
```

Comment:
This is always going to be falling out of style over time.  Should we just start at 0?

Reply:
🤔